src/components/redux/bookForm.js

```diff
@@ -1,8 +1,8 @@
 import { nanoid } from '@reduxjs/toolkit';
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
-import { addBook, postBooks } from '../books/booksSlice';
-import './styles/nav.css';
+import { addBook, postBooks } from './books/booksSlice';
```

Comment:
**You did a great job!**

- [ ] But, as the previous reviewer showed in the example, your `redux` must be alongside the components folder not inside. Please fix it and resubmit it again. Thank you!